package.json

```diff
@@ -40,10 +40,13 @@
     "webidl-conversions": "^7.0.0",
     "whatwg-encoding": "^2.0.0",
     "whatwg-mimetype": "^3.0.0",
-    "whatwg-url": "^12.0.1",
+    "whatwg-url": "^13.0.0",
     "ws": "^8.13.0",
     "xml-name-validator": "^4.0.0"
   },
+  "resolutions": {
+    "whatwg-url": "^13.0.0"
+  },
```

Comment:
This resolutions is to say to yarn that even if `data-urls` asks the vesion `^12.0.0`, we should use the version `^13.0.0`

I can remove it but this would create a duplication of the library `whatwg-url` and inconstancies with URL manipulations in different places for JSDOM